recipes/pyspark-stubs/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pyspark-stubs" %}
+{% set version = "2.4.0.post5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "2249e20f3d120551d6ba3bf8bca67c1cb9a2d2e39bd9ebcd3cb00ae906428f21"
```

Could you please remove the quotes from ``sha256``?
```suggestion
  sha256: 2249e20f3d120551d6ba3bf8bca67c1cb9a2d2e39bd9ebcd3cb00ae906428f21
```